### PR TITLE
Fix CI var and comply with new Prettier version

### PR DESCRIPTION
**Pull request by rick-stevens**

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "ESLINT_EXTENDS": {
     "none": "[\"airbnb-base\", \"plugin:prettier/recommended\"]",
     "react": "[\"airbnb\", \"plugin:prettier/recommended\"]",
-    "vue": "[\"plugin:vue/recommended\", \"airbnb-base\", \"prettier/vue\", \"plugin:prettier/recommended\"]"
+    "vue": "[\"plugin:vue/recommended\", \"airbnb-base\", \"prettier\", \"plugin:prettier/recommended\"]"
```
Comment (rick-stevens) [inline src/config.json:27]: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

File: src/templates/default/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "{{dagStartScript}}start": "echo 'No start specified'",
     "download-env": "GITLAB_GROUP={{gitlabNamespace}} tools/envhelper download -p {{appName}}",
     "upload-env": "GITLAB_GROUP={{gitlabNamespace}} tools/envhelper upload -p {{appName}}",
-    "postinstall": "if [ \"$CI_SERVER\" != 'yes' ]; then curl -o ./tools/envhelper --create-dirs https://cdn.greenhousegroup.com/ghg-nl/meh/binaries/envhelper-latest && chmod +x ./tools/envhelper; fi"
+    "postinstall": "if [ -z \"$CI_SERVER_URL\" ]; then curl -o ./tools/envhelper --create-dirs https://cdn.greenhousegroup.com/ghg-nl/meh/binaries/envhelper-latest && chmod +x ./tools/envhelper; fi"
```
Comment (rick-stevens) [inline src/templates/default/package.json:13]: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html#predefined-variables-reference

File: src/config.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
   "ESLINT_EXTENDS": {
     "none": "[\"airbnb-base\", \"plugin:prettier/recommended\"]",
     "react": "[\"airbnb\", \"plugin:prettier/recommended\"]",
-    "vue": "[\"plugin:vue/recommended\", \"airbnb-base\", \"prettier/vue\", \"plugin:prettier/recommended\"]"
+    "vue": "[\"plugin:vue/recommended\", \"airbnb-base\", \"plugin:prettier/recommended\"]"
```
Comment (rick-stevens) [inline src/config.json:27]: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

File: src/actions/createFiles.js [hunk 4]
```diff
@@ -96,11 +96,13 @@ module.exports = (answers) => {
...
-    fileName: transformFileName ? transformFileName(file.fileName) : file.fileName,
-    fileContents: file.fileContents.replace(/{{([^}]+)}}/g, (_, match) => data[match] || ''),
-  });
+  const overrideContents =
+    (transformFileName = undefined) =>
+    (file) => ({
+      ...file,
+      fileName: transformFileName ? transformFileName(file.fileName) : file.fileName,
+      fileContents: file.fileContents.replace(/{{([^}]+)}}/g, (_, match) => data[match] || ''),
+    });
```
Comment (rick-stevens) [inline src/actions/createFiles.js:105]: Auto lint fix after Prettier update

**Review by LukevdPalen — APPROVED** [review]

**Pull request merged**